ketchup/main.py

```diff
@@ -37,35 +37,37 @@ def run():
         try:
             episodes = int(
                 input(
-                    "Hello! How many episodes are in the show you are interested in watching?: "
+                    "\nHello! How many episodes are in the show you are interested in watching?:\n"
                 )
             )
         except ValueError:
-            print("Sorry, please only enter a value greater than 0")
+            print("Sorry, please only enter a value greater than 0.")
             continue
 
         if episodes <= 0:
-            print("Sorry, please only enter a value greater than 0")
+            print("Sorry, please only enter a value greater than 0.")
             continue
         else:
             break
 
     end_date_prompt = (
-        "What date would you like to have completed this series?(ex: 15 Aug 2050): "
+        "What date would you like to have completed this series?(Ex: 15 Aug 2050):\n"
     )
     end_date = get_date(end_date_prompt, require_future=True)
 
-    answer_today = input("Will you start watching today? [Y/n]: ") or "y"
+    answer_today = input("Will you start watching today? [Y/N]:\n") or "y"
     if answer_today.lower() == "y":
         start_date = dt.datetime.today()
     else:
-        start_date_prompt = "What day will you start?(15 Aug 2023): "
+        start_date_prompt = "What day will you start? (Ex: 15 Aug 2023):\n"
         start_date = get_date(start_date_prompt)
 
-    per_day_limit = (
-        input("What is the limit of episodes you will watch per day? [0+] ") or 0
+    per_day_limit = int(
+        input("What is the limit of episodes you will watch per day? [0+]\n") or 0
     )
 
+    days_per_week_watching = int(input("How many days a week do you plan to watch?:\n"))
```

Comment:
perhaps we need an explainer here?

"Some people may not be able to watch 7 days a week, how many days per week will you be available to watch episodes?" or something similar? thoughts?

Reply:
Trying to find a way to offer that without it being too verbose.

Maybe a user input to be taken before this one. Asking the user if they can/cannot dedicate time to watching an episode a day, and if they can't dedicate an episode daily, the user input for days_per_week_watching would be prompted.